css/css-backgrounds/border-image-beats-transparent-border-color-ref.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>
+  border-image should beat transparent border-color
```

Comment:
"beat" is a bit odd here. Maybe "border-color: transparent should not hide the border-image"